README.md

```diff
@@ -243,20 +243,17 @@ dynamics = Dynamics(DynamicsFcn.TORQUE_DRIVEN)
 ```
 
 The pendulum is required to start in a downward position (0 rad) and to finish in an upward position (3.14 rad) with no velocity at start and end nodes.
-To define that, it would be nice to first define boundary constraints on the position (*q*) and velocities (*qdot*) that match those in the bioMod file and to apply them at the very beginning, the very end and all the intermediate nodes as well.
+In this case, the degree of freedom with index 0 is translation y, and the index 1 refers to rotation x. 
```

Comment:
rotation about x

Reply:
The `state` instead of degree of freedom, as indices refer to the state variable (where the first part is the degree and the second is obviously not!)